tools/cockpit.spec

```diff
@@ -189,6 +207,13 @@ install -p -m 644 tools/cockpit.pam $RPM_BUILD_ROOT%{_sysconfdir}/pam.d/cockpit
 rm -f %{buildroot}/%{_libdir}/cockpit/*.so
 install -D -p -m 644 AUTHORS COPYING README.md %{buildroot}%{_docdir}/cockpit/
 
+# selinx
+%if 0%{?with_selinux}
+    install -D -m 644 %{name}.pp.bz2 %{buildroot}%{_datadir}/selinux/packages/%{selinuxtype}/%{name}.pp.bz2
```

Comment:
It feels a bit weird having to install these from the spec only, instead of from `make install`. But good enough for now, especially while we don't ship these in Debian and don't build the manpage from XML/md